third_party/terraform/resources/resource_cloudfunctions_function.go

```diff
@@ -117,7 +101,7 @@ func resourceCloudFunctionsFunction() *schema.Resource {
 				Type:         schema.TypeString,
 				Required:     true,
 				ForceNew:     true,
-				ValidateFunc: validateResourceCloudFunctionsFunctionName,
```

Comment:
I think we still want to use this? Since it says above it has different caps rules

Reply:
yuppp forgot to copy this back to mm